evap/evaluation/models.py

```diff
@@ -31,6 +32,8 @@
 logger = logging.getLogger(__name__)
 
 
+
+
```

Comment:
I think we want 2 empty lines here, so I'd remove these again (very interesting that codacy doesn't complain here, I don't think that this is pep8 compliant)